<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Graficos">
     <attribute name="title">
      <string>Gráficos</string>
     </attribute>
     <widget class="QGraphicsView" name="graficos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="dibujar_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dibujar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="limpiar_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tabla">
     <attribute name="title">
      <string>Tabla</string>
     </attribute>
     <widget class="QTableWidget" name="tabla">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="buscar_line_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID Particula</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscar_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>510</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mostrar_tabla_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>510</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Ordenar por:</string>
      </property>
      <widget class="QCheckBox" name="id_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="distancia_checkbox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Distancia</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="velocidad_checkbox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Velocidad</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Color RGB:</string>
    </property>
    <widget class="QSpinBox" name="red">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>-30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Color (RGB):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="blue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="green">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Partículas</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Coordenadas de Origen</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="origen_X">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="origen_y">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Coordenadas de Destino</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="destino_x">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="destino_y">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Velocidad</string>
    </property>
    <widget class="QSpinBox" name="vel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vel:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="agregar_final_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregar_inicio_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Inicio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mostrar_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>610</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generar_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>610</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGrafo"/>
    <addaction name="actionRecorrido"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionGrafo">
   <property name="text">
    <string>Generar grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Amplitud">
   <property name="text">
    <string>en Amplitud</string>
   </property>
  </action>
  <action name="action_Profundidad">
   <property name="text">
    <string>en Profundidad</string>
   </property>
  </action>
  <action name="actionRecorrido">
   <property name="text">
    <string>Recorrido de grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
